docs/sources/configure-client/go-span-profiles.md

```diff
@@ -64,37 +78,25 @@ func main() {
 }
 ```
 
-Now that we set up the tracer, we can create a new trace from anywhere and the profiler will automatically capture profiles for it.
+Now that you set up the tracer, you can create a new trace from anywhere and the profiler automatically captures profiles for it.
 ```go
 ctx, span := otel.Tracer("tracerName").Start(ctx, "ExampleSpan")
 defer span.End()
 
 // Your code goes here.
 ```
 
-## View the span profiles in Grafana Tempo
-
-To view the span profiles in Grafana Tempo, you need to have a Grafana instance running and a data source configured to link trace spans and profiles.
-
-Refer to the [data source configuration documentation](/docs/grafana/datasources/tempo/configure-tempo-data-source) to see how to configure the visualization to link trace spans with profiles.
-
-To use a simple configuration, follow these steps:
-
-1. Select a Pyroscope data source from the Data source drop-down.
-
-2. Optional: Choose any tags to use in the query. If left blank, the default values of service.name and service.namespace are used.
-
-The tags you configure must be present in the spans attributes or resources for a trace to profiles span link to appear. You can optionally configure a new name for the tag. This is useful for example if the tag has dots in the name and the target data source doesn’t allow using dots in labels. In that case you can for example remap service.name to service_name.
+## View the span profiles in Grafana or Grafana Cloud
 
```

Comment:
We have the full content for Traces to profiles in the Pyroscope documentation. I removed the content from the Traces to profiles and linked to the page instead. 

Doc is here: https://grafana.com/docs/pyroscope/next/view-and-analyze-profile-data/profile-tracing/traces-to-profiles/ 

We can make it live. It's currently in /next. Not sure why it isn't live.